File: api-server-bis/models/room.js
module.exports = function (sequelize, DataTypes) {
  const Room = sequelize.define('Room', {
    name: {
      type: DataTypes.STRING,
      validate: {
        notEmpty: true
      }
    },
    organisation: {
      type: DataTypes.STRING,
      validate: {
        notEmpty: true
      }
    },
    volume: {
      type: DataTypes.INTEGER,
      validate: {
        min: 0,
        isInt: true
      }
    },
    realLight: {
      type: DataTypes.INTEGER,
      validate: {
        min: 0,
        isInt: true
      }
    },
    light: {
      type: DataTypes.INTEGER,
      validate: {
        min: 0,
        isInt: true
      }
    },
    maxLux: {
      type: DataTypes.INTEGER,
      validate: {
        min: 0,
        isInt: true
      }
    },
    realTemperature: {
      type: DataTypes.INTEGER,
      validate: {
        min: 0,
        isInt: true
      }
    },
    temperature: {
      type: DataTypes.INTEGER,
      validate: {
        min: 0,
        isInt: true
      }
    },
    artificialIntellligence: {
      type: DataTypes.BOOLEAN,
      defaultValue: false
    },
    data: DataTypes.JSON,
    planning: DataTypes.ARRAY(DataTypes.JSON),
    m: {
      type: DataTypes.FLOAT,
      validate: {
        isFloat: true
      }
    },
    off: {
      type: DataTypes.FLOAT,
      validate: {
        isFloat: true
      }
    }
  }, {
    classMethods: {
      associate(models) {
        Room.hasMany(models.Window, { as: 'windows', foreignKey: 'roomId' });
        Room.hasMany(models.AirConditioning, { as: 'airConditionings', foreignKey: 'roomId' });
        Room.hasMany(models.Captor, { as: 'captors', foreignKey: 'roomId' });
      },
    },
  });

  return Room;
};

File: Windows phone app/EnvioApp/EnvioApp/Room.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EnvioApp
{
    class Room
    {
        [JsonProperty(PropertyName = "name")]
        public string name { get; set; }
        [JsonProperty(PropertyName = "_id")]
        public string _id { get; set; }
        [JsonProperty(PropertyName = "organisation")]
        public string organisation { get; set; }

//        [JsonProperty(PropertyName = "planning")]
        public List<Planning>   planning { get; set; }
        [JsonProperty(PropertyName = "temperature")]
        public string temperature { get; set; }
        [JsonProperty(PropertyName = "realTemperature")]
        public string realTemperature { get; set; }
        [JsonProperty(PropertyName = "light")]
        public string light { get; set; }
        [JsonProperty(PropertyName = "realLight")]
        public string realLight { get; set; }

        public override string ToString()
        {
            return (name);
/*            return ("ID: " + _id + " - Name: " + name + " - Organisation: " + organisation +
                " - temperature: " + temperature + " - Light: " + light);
*/
        }
    }
}
